cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -249,7 +249,7 @@ message SearchJobStatusRequest {
         }
     }
 
-    Filter filter = 1;
+    repeated Filter filter = 1;
```

Comment:
Will these filters always be anded together? Might be worth clarifying in some way.

Reply:
Yeah they'll always be anded together, I can add a comment